niftly.pro

```diff
@@ -1,10 +1,15 @@
 TEMPLATE = app
 
-QT += qml quick svg
+QT += qml quick svg network
 
 SOURCES += main.cpp
 
-RESOURCES += qml.qrc
+RESOURCES += qml.qrc \
```

Comment:
IIRC, there is no need for that. All qt libs and QML modules are automatically included in apk bundle when packaging the application. Did you get any error with original niftly code?